### PR TITLE
Add dropdown/select update functionality

### PR DESCRIPTION
**Pull request by nshontz**
I needed a way to change the value of a form dropdown, I noticed I could update text input fields, and so I modeled the select update on the 'types' functionality. There was already a 'selects' method so I named the function 'dropdown_select'

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you also update the readme with an example of the new functionality?

File: README.md [hunk 1]
```diff
@@ -584,6 +584,26 @@ Browsershot::url('https://example.com')
...
+Browsershot::url('https://example.com')
+    ->dropdown_select('#selector1', '100')
+```
+
+You can combine `dropdown_select`, `type` and `click` to create a screenshot of a page after submitting a form:
+
+```php
+Browsershot::url('https://example.com')
+    ->type('#firstName', 'My name')
+    ->dropdown_select('#state', 'MT')
```
Comment (freekmurze) [inline README.md]: rename the method to `select`

File: src/Browsershot.php [hunk 2]
```diff
@@ -146,6 +146,17 @@ public function click(string $selector, string $button = 'left', int $clickCount
         return $this;
     }
 
+    public function dropdown_select(string $selector, string $value = '')
+    {
+        $dropdown_selects = $this->additionalOptions['selects'] ?? [];
```
Comment (freekmurze) [inline src/Browsershot.php]: Make this variable camelCased.

File: README.md (hunk 1 above)
Comment (nshontz) [reply README.md]: There is already a method named [select](https://github.com/spatie/browsershot/blob/master/src/Browsershot.php#L215) I'm not in love with dropdown_select, but couldn't think of a good descriptive name.

Comment (danhunsaker) [reply README.md]: `select_option`?

Comment (freekmurze) [reply README.md]: `selectOption` sounds good

Comment (nshontz) [reply README.md]: Thanks! It is done!

File: src/Browsershot.php (hunk 2 above)
Comment (nshontz) [reply src/Browsershot.php]: This has also been changed.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**